lib/logstash/inputs/kinesis.rb

```diff
@@ -53,6 +53,9 @@ class LogStash::Inputs::Kinesis < LogStash::Inputs::Base
   # Select AWS profile for input
   config :profile, :validate => :string
 
+  # Select initial_position_in_stream. Accepts TRIM_HORIZON or LATEST
+  config :initial_position_in_stream, :validate => :string, :default => "TRIM_HORIZON"
```

Comment:
let's change `:validate => :string` to `:validate => ["TRIM_HORIZON", "LATEST"]`, this way they can't enter some other random string and end up still getting LATEST due to the conditional below. See the `:metrics` config var above for an example.